spark-jobs/src/test/scala/za/co/absa/enceladus/conformance/streaming/HyperConformanceMappingIntegrationSuite.scala

```diff
@@ -13,28 +13,26 @@
  * limitations under the License.
  */
 
-//todo reenable with Hyperdrive API for Scala 2.12 #1712
+package za.co.absa.enceladus.conformance.streaming
 
+import org.apache.spark.sql.DataFrame
+import org.scalatest.funsuite.AnyFunSuite
+import za.co.absa.enceladus.conformance.interpreter.fixtures.{MultipleMappingFixture, StreamingFixture}
 
-//package za.co.absa.enceladus.conformance.streaming
-//
-//import org.apache.spark.sql.DataFrame
-//import org.scalatest.funsuite.AnyFunSuite
-//import za.co.absa.enceladus.conformance.interpreter.fixtures.{MultipleMappingFixture, StreamingFixture}
-//
-//class HyperConformanceMappingIntegrationSuite extends AnyFunSuite with StreamingFixture with MultipleMappingFixture {
-//
-//  test("Test streaming multiple mapping") {
-//    implicit val infoDateFactory: InfoDateFactory = new InfoDateLiteralFactory("2020-05-23")
-//    val df: DataFrame = testHyperConformance(standardizedDf,
-//      "result", mappingDS)
-//      .orderBy("result.property")
-//
-//    assertResult(3)(df.count())
-//    val conformed = spark.read
-//      .textFile("src/test/testData/multipleMapping/conformed_multiple_mapping.json")
-//      .collect().mkString("\n")
-//    val returned = df.toJSON.collect().mkString("\n")
-//    assertResult(returned)(conformed)
-//  }
-//}
+class HyperConformanceMappingIntegrationSuite extends AnyFunSuite with StreamingFixture with MultipleMappingFixture {
+
+  // todo dependent on resolving #1744 udf for spark 3
```

Comment:
What about this TODO?

Reply:
This means that it can only be enabled (`ignore` -> `test`) after #1744 is merged in.

Reply:
Believe this can be now reverted.

Reply:
Done.